kafka-client/src/main/java/dev/responsive/kafka/internal/utils/StoreUtil.java

```diff
@@ -92,6 +93,17 @@ public static long durationToMillis(final Duration duration, final String parame
     }
   }
 
+  public static String streamThreadId() {
+    final String threadId = Thread.currentThread().getName();
+    final var regex = Pattern.compile(".*-(StreamThread-\\d+)");
```

Comment:
ditto as above, why not just use the entirety of the thread name?

Reply:
yeah I guess we could. It's a bit more of a change and it was late and I was tired :)